src/backend/packages/inensus/textbee-sms-gateway/src/resources/assets/js/app.js

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Can you remove all the `src/resources` files from backend code? They don't belong here